src/Extensions/ElementalPageExtension.php

```diff
@@ -19,4 +21,21 @@ class ElementalPageExtension extends ElementalAreasExtension
     private static $owns = [
         'ElementalArea'
     ];
+
+    public function MetaTags(&$tags)
+    {
+        $controller = Controller::curr();
+        $request = $controller->getRequest();
+        if ($request->getVar('ElementalPreview') !== null) {
+            $html = HTML4Value::create($tags);
+            $xpath = "//meta[@name='x-page-id' or @name='x-cms-edit-link']";
+            $removeTags = $html->query($xpath);
+            $body = $html->getBody();
+            foreach ($removeTags as $tag) {
+                $body->removeChild($tag);
+            }
+            $tags = $html->getContent();
+        }
+        return $tags;
```

Comment:
we probably don't need to return here since it's modified by reference

Reply:
It's a tidy habit of mine as a while back it was introduced that you could do either (return or reference). But without the reference declaration here it no longer works... so yeah; it is pointless.